service/db.hpp

```diff
@@ -42,18 +48,21 @@ namespace raven
   inline constexpr const db_statement_st update_config_text_from_id_statement{
       R"(UPDATE config set config_text = ? where id = ?;)"};
 
-  struct config_create_answer_db
+  // maybe do fieldbits in the future, but seems sufficient for now
+  enum class db_state : short
   {
-    config_key_st config_key;
-    config_key_st readonly_config_key;
-    config_id_st config_id;
-    std::optional<request_state> state_error;
+    good,
```

Comment:
```suggestion
    ok,
```
or
```suggestion
    no_error,
```
seem like a more common way to name such a value

Reply:
I named it that way because the function to check if the operation left the state of the db viable, but I can change it to ok if you think this isn't a fitting name

Reply:
I'd at least explicitly put it equal to be 0. I've just never seen "good" used for such a value before.

Reply:
I will change it to ok it'll be clearer